bitnami/tomcat/Chart.yaml

```diff
@@ -26,4 +26,4 @@ name: tomcat
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/tomcat
   - http://tomcat.apache.org
-version: 10.6.1
+version: 10.6.2
```

Comment:
The change you propose is adding new variables, therefore it is not fixing a simple bug. It would be more appropriate, following the [semver.org ](https://semver.org) guidelines, for the version to increase in a minor and not a patch.
Therefore, the value should be 10.7.0.